nic: clarify config-safety (#1857)
nginx/documentation: content/nic/configuration/global-configuration/command-line-arguments.md

### -enable-config-safety

> {{< call-out "caution" >}}This is an experimental feature. Behavior and configuration may change in future releases. When enabled, users can experience delayed pod startup times as resources are validated prior to being written, and delays will be directly proportional to number and complexity of resources.  {{< /call-out >}}

Reviewer: ```suggestion
{{< call-out "caution" >}}This is an experimental feature. Behavior and configuration may change in future releases. When this feature is turned on, users can experience delayed pod startup times as resources are validated prior to being written. Delays are directly proportional to the number and complexity of resources.  {{< /call-out >}}
```

> The `-enable-config-safety` flag enhances the stability and reliability of the NGINX Ingress Controller. When enabled, it ensures the validation of new configurations before reloading nginx. If a configuration is deemed invalid, the new config for associated resource is rejected and old working config is restored. By Default it is set to `false`. 

Reviewer: ```suggestion
The `-enable-config-safety` flag enhances the stability and reliability of the NGINX Ingress Controller. When turned on, this feature ensures the validation of new configurations before reloading nginx. If a configuration is deemed invalid, the new config for the associated resource is rejected and the old working config is restored. By Default it is set to `false`. 
```
